bigbluebutton-html5/imports/ui/components/plugins-engine/container.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useEffect, useRef, useState, useMemo } from 'react';
+import logger from '/imports/startup/client/logger';
+
+import PluginHooksHandlerContainer from './plugin-hooks-handler/container';
+import PluginsEngineComponent from './component';
+import { PluginConfigSetting, PluginEngineProps, UniquePluginConfigSetting } from './types';
+import PluginsLoaderComponent from './plugin-loader/component';
+import PluginProvidedStateComponent from './plugin-provided-state/component';
+
+const PLUGINS = Meteor.settings.public.plugins;
+
+const createUuid = () => {
+  const UUID = "xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx".replace(/[xy]/g, (c) => {
+      const r = Math.floor(Math.random() * 16);
+      const v = c === "x" ? r : (r % 4) + 8;
+      return v.toString(16);
+  });
+  return UUID;
+}
```

Comment:
Use an actual uuid library instead of this implementation.

See: https://www.npmjs.com/package/uuid